roles/openstack/neutron/tasks/cleanup.yml

```diff
@@ -0,0 +1,30 @@
+---
+# This file is used to cleanup OpenStack neutron resources
+
+- name: Cleanup routers
+  vars:
+      prefix: "{{ provisioner.prefix | default(uuid) }}-"
```

Comment:
Why not move the prefix to defaults? and apply it to all names at once? instead of setting it as task var?
You could also put everything in a block and define block vars (same result).